package.json

```diff
@@ -15,9 +15,9 @@
     "lint": "yarn lint:js",
     "lint:js": "eslint .",
     "start": "ember server --no-live-reload --port=0",
-    "test": "yarn test:ember && yarn test:node",
+    "test": "yarn test:ember",
     "test:ember": "ember test",
-    "test:node": "qunit node-tests/run.js"
+    "test:dependency-management": "node dependency-tests/test-all.js"
```

Comment:
did node-tests go away?

Reply:
yes

Reply:
The previous node-tests existed for the previous dependency-management solution which has been removed.